lib/tasks/assets.rake

```diff
@@ -0,0 +1,9 @@
+if Rails.env.production?
```

Comment:
Nope, this piece of code prevents asset precompilation from the production environment.